### DISCUSSION TITLE
夸克速度慢

### DISCUSSION THREAD

**Discussion by thomasdba** [AlistGo/alist#2173] [General]
通过webdav挂载速度卡住上不去，无法满足4K在线观看，但如果用官方客户端下载，速度可以达到40MB/s，是大B。所以有些好奇，什么原因，是否可以改进，谢谢！

**Comment by LeeeSe** [comment]
貌似有人说需要改成夸克的UA
暂时没有大佬愿意明确公开方法

**Comment by zjb19981222** [comment]
是的，我看视频一直转圈圈

**Comment by anwen-anyi** [comment]
夸克墙强制理模式， 你Alist搭建在服务器  除非你服务器的宽带很大.......要么搭建在本地或者路由器里面，要么别想了

**Reply by thomasdba** [reply]
我的alist就是搭在软路由上啊，一样很慢，奇怪

**Reply by kuko7** [reply]
我是用安卓手机搭建的alist，为什么速度还是慢呢？

**Comment by xt9058** [comment]
因为夸克网盘的限制。大文件需要客户端下载。而真实地址也只能等入进了网页才能获得。所以我从alist里面获得都是本地中转过来的数据。带宽也是取决你本地上传的带宽。这个与目前的115网盘是一样的。所以等非官方接口开发出来。像破解百度网盘那种吧

**Reply by thomasdba** [reply]
我在家里软路由跑的alist

**Comment by rakaki** [comment]
夸克我也是，串流播放速度只有2m。不过用mixplorer配合mxplayer的话，速度倒是能达到6m左右。直接在网页alist里面下载的话速度也很快

**Comment by github123werwtwewqe** [comment]
夸克网盘挂载了，但是无法播放视频，该怎么办？

**Reply by github123werwtwewqe** [reply]
token也重新更新输了一遍，但打开视频就显示重新连接，然后错误

**Reply by github123werwtwewqe** [reply]
有人知道怎么办吗？🤔

**Reply by terry9407** [reply]
一样的问题

**Comment by alanhe421** [comment]
除了阿里云盘有别的不慢的吗

**Reply by CharlesZHEN** [reply]
天翼云超级快

**Comment by thomasdba** [comment]
这个问题已经在最新版本里解决了，挺好，这样夸克还挺值得

**Reply by dingodingle** [reply]
还是很慢啊，基本不能看，什么好办法

**Reply by Nothingbutfish** [reply]
我就只有100多kb的速度。。

**Reply by verydemo** [reply]
解决了么？ 我用的v3.41.0， 有88vip,速度只有100K

**Comment by qcnhy** [comment]
TV也限制死了 只有100k速度

**Reply by alanhe421** [reply]
走夸克云盘直接看吧，我现在是AirPlay云盘去看

**Reply by qcnhy** [reply]
<img width="450" height="331" alt="image" src="https://github.com/user-attachments/assets/2163feb5-117d-4378-885c-f0b8a2b0c16e" />
破防

**Reply by hansostong** [reply]
是啊，虽然是TV版是直链播放，但速度锁死100k

**Reply by wawebo** [reply]
<img width="1920" height="1080" alt="58d09778-5897-4f53-9659-de4be85be30f" src="https://github.com/user-attachments/assets/a42f7675-d1cc-4922-a4dd-0b68b883647b" />
奇怪我用 IDM 下载就满速

**Comment by qcnhy** [comment]
那普通的电视机安装tv版本也锁100k？

________________________________
发件人: ***@***.*** ***@***.***> 代表 hansostong ***@***.***>
发送时间: 2025年9月8日 11:38
收件人: AlistGo/alist ***@***.***>
抄送: qcnhy ***@***.***>; Comment ***@***.***>
主题: Re: [AlistGo/alist] 夸克速度慢 (Discussion #2173)


是啊，虽然是TV版是直链播放，但速度锁死100k

―
